### PR TITLE
clear code

### PR DESCRIPTION
**Pull request by dorosh**

### CODE REVIEW

File: video_xblock/static/css/brightcove.css [hunk 1]
```diff
@@ -326,4 +326,11 @@ body {
     color: inherit;
     background: rgba(115,133,159,.5);
     border-radius: 0;
-}
\ No newline at end of file
+}
+/* styles for theme */
+.bc-player-default_default .vjs-big-play-button:active,
```
Comment (z4y4ts) [inline video_xblock/static/css/brightcove.css]: Please add a comment about `default_default` and Brightcove's themes.
So we can find this in the future.

File: video_xblock/video_xblock.py [hunk 2]
```diff
@@ -185,6 +185,9 @@ def save_player_state(self, request, suffix=''):
         return {'success': True}
 
     def clean_studio_edits(self, data):
+        """
+        Helper method for change player_name into data.
```
Comment (z4y4ts) [inline video_xblock/video_xblock.py]:
```python
"""
Given POST data dictionary 'data', clean the data before validating it.

Tries to detect player by submitted video url. If fails, it defaults to 'dummy-player'
"""
```

Comment (dorosh) [inline video_xblock/video_xblock.py]: ok

File: video_xblock/static/css/brightcove.css (hunk 1 above)
Comment (dorosh) [inline video_xblock/static/css/brightcove.css]: ok

**Pull request merged**